SINFONI/Context.cs

```diff
@@ -101,6 +101,14 @@ public void OpenConnection(string configURI, Action<Connection> onConnected)
                 establishedConnection.LoadIDL(ServerConfiguration);
                 onConnected(establishedConnection);
             };
+            try
+            {
+                transportConnection.Open();
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("Failed to open Socket connection to ws://{0}:{1}, Reason: {2} ", host, port, e.Message);
```

Comment:
Exception message should be generic and not mention particular transport methods, e.g. socket.